NTupleMaker/src/Synch17GenTree.cc

```diff
@@ -160,34 +178,52 @@ void Synch17GenTree::WriteInit(TTree *tree) {
   fChain = tree;
   fCurrent = -1;
 
-   fChain->Branch("Higgs_pt", &Higgs_pt, "Higgs_pt/F");
-   fChain->Branch("Higgs_eta", &Higgs_eta, "Higgs_eta/F");
-   fChain->Branch("Higgs_phi", &Higgs_phi, "Higgs_phi/F");
-   fChain->Branch("Higgs_mass", &Higgs_mass, "Higgs_mass/F");
-
-   fChain->Branch("genpt_1", &genpt_1, "genpt_1/F");
-   fChain->Branch("genphi_1", &genphi_1, "genphi_1/F");
-   fChain->Branch("geneta_1", &geneta_1, "geneta_1/F");
-   fChain->Branch("genmode_1", &genmode_1, "genmode_1/I");
-
-   fChain->Branch("genpt_2", &genpt_2, "genpt_2/F");
-   fChain->Branch("genphi_2", &genphi_2, "genphi_2/F");
-   fChain->Branch("geneta_2", &geneta_2, "geneta_2/F");
-   fChain->Branch("genmode_2", &genmode_2, "genmode_2/I");
-
-   fChain->Branch("acotautau_00", &acotautau_00, "acotautau_00/F");
-   fChain->Branch("acotautau_10", &acotautau_10, "acotautau_10/F");
-   fChain->Branch("acotautau_01", &acotautau_01, "acotautau_01/F");
-   fChain->Branch("acotautau_11", &acotautau_11, "acotautau_11/F");
-   fChain->Branch("acotautau_02", &acotautau_02, "acotautau_02/F");
-   fChain->Branch("acotautau_12", &acotautau_12, "acotautau_12/F");
-   fChain->Branch("acotautau_20", &acotautau_20, "acotautau_20/F");
-   fChain->Branch("acotautau_21", &acotautau_21, "acotautau_21/F");
-   fChain->Branch("acotautau_22", &acotautau_22, "acotautau_22/F");
-
-   fChain->Branch("GenVertexX", &GenVertexX, "GenVertexX/F");
-   fChain->Branch("GenVertexY", &GenVertexY, "GenVertexY/F");
-   fChain->Branch("GenVertexZ", &GenVertexZ, "GenVertexZ/F");
+   fChain->Branch("GEN_Higgs_pt", &Higgs_pt, "GEN_Higgs_pt/F");
```

Comment:
I have no idea of why you changed the names of the variables displayed in the gen level tree... Would you mind explaining it to me?

Reply:
Hi Andrea, please see comment above ;-)